packages/client/hmi-client/src/components/Navbar.vue

```diff
@@ -116,120 +176,81 @@ watch(activeProjectName, (newProjectName) => {
 watch(currentRoute, (newRoute) => {
 	selectedPage.value = navItems.value[newRoute.path] || emptyNavItem;
 });
-</script>
 
-<template>
-	<header>
-		<section class="header-left">
-			<img src="@assets/svg/terarium-logo.svg" height="48" width="168" alt="TERArium logo" />
-			<nav>
-				<Dropdown
-					class="dropdown"
-					v-model="selectedPage"
-					:options="Object.values(navItems)"
-					optionLabel="name"
-					panelClass="dropdown-panel"
-					@change="goToPage"
-				>
-					<template #value="slotProps">
-						<i :class="slotProps.value.icon" />
-						<span>{{ slotProps.value.name }}</span>
-					</template>
-					<template #option="slotProps">
-						<i :class="slotProps.option.icon" />
-						<span>{{ slotProps.option.name }}</span>
-					</template>
-				</Dropdown>
-			</nav>
-		</section>
-		<SearchBar
-			class="searchbar"
-			:text="searchBarText"
-			:suggested-terms="relatedSearchTerms"
-			@search-text-changed="searchTextChanged"
-			@toggle-search-by-example="searchByExampleModalToggled"
-			:resultType="resultType"
-		/>
-		<section class="header-right">
-			<Button
-				class="p-button p-button-icon-only p-button-rounded p-button-sm user-button"
-				@click="showUserMenu"
-			>
-				{{ userInitials }}
-			</Button>
-		</section>
-		<Menu ref="userMenu" :model="userMenuItems" :popup="true"> </Menu>
-		<Dialog header="Logout" v-model:visible="isLogoutConfirmationVisible">
-			<span>You will be returned to the login screen.</span>
-			<template #footer>
-				<Button label="Ok" class="p-button-text" @click="auth.logout"></Button>
-				<Button
-					label="Cancel"
-					class="p-button-text"
-					@click="isLogoutConfirmationVisible = false"
-				></Button>
-			</template>
-		</Dialog>
-	</header>
-</template>
+/*
+ * Search
+ */
+const terms = ref<string[]>([]);
+
+async function queryChanged(q: string | null) {
+	// Empty the related terms when the query is over
+	if (!q) terms.value = [];
```

Comment:
In this pr: https://github.com/DARPA-ASKEM/TERArium/pull/615/files I made `getRelatedWords` just return an empty array if the term is blank, so we won't need to do these conditional checks.

Reply:
I agree, but I know from that point on that if the query is empty I can return an empty array.
So better do it here that trusting a function that might change later.